src/components/sections/Cards/RitcherCard.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+type Level = {
+  id: string;
+  range: string;
+  label: string;
+  description: string;
+  color: string;
+};
+
+const levels: Level[] = [
+  {
+    id: 'light',
+    range: '2.0 a 3.0',
+    label: 'Ligero',
+    description:
+      'Puede estar entre 2.0 a 3.0 en la escala Ritcher, no es percibido ni causa daños en donde fue originado.',
+    color: 'bg-[#86FE8B]',
+  },
+  {
+    id: 'moderate',
+    range: '3.0 a 5.0',
+    label: 'Moderado',
+    description:
+      'Puede estar entre 3.0 a 5.0 en la escala Ritcher, podria ser percibido cerca de tu ubicación.',
+    color: 'bg-[#F6F971]',
+  },
+  {
+    id: 'strong',
+    range: '5.0 a 7.0',
+    label: 'Fuerte',
+    description:
+      'Puede estar entre 5.0 a 7.0 en la escala Ritcher, puede ser peligroso y va a ser percibido en tu ubicación.',
+    color: 'bg-[#FEAD86]',
+  },
+  {
+    id: 'extreme',
+    range: '7.0 en adelante',
+    label: 'Extremo',
+    description:
+      'Puede estar entre 7.0 en adelante en la escala Ritcher, es peligroso y causa grandes daños en distintos puntos.',
+    color: 'bg-[#FE8686]',
+  },
+];
+
+function RichterScale() {
```

Comment:
5. Let's rename the component to `RitcherScaleSection` to follow the convention of having the same name as the file but in `CamelCase`